README.md

```diff
@@ -59,16 +63,24 @@ This repository contains helper scripts and commands to create meetup videos for
 
         $ ffmpeg -i joined.mp4 -i final.wav -shortest -preset ultrafast -q 0 final.mp4
 
-* scale video to 16:9 
+* Managing volume in the video
+
+        $ ffmpeg -i <input_video_name> -af 'volume=2.5<or 1x>' -preset ultrafast <output_video_name>
+
+* scale video to 16:9
 
         $ ffmpeg -i final.mp4 -vf pad="ih*16/9:ih:(ow-iw)/2:(oh-ih)/2" -preset ultrafast -acodec copy final2.mp4
 
-* Generate the title image 
+    or specify absolute pixels
+
+        $ ffmpeg -i <input_video_name> -vf "scale=<specify_width_in_px:specify_length_in_px>" -preset ultrafast <output_video_name>
+
```

Comment:
I'm still don't know why the scaling is required, what are trying to solve/introduce here ? its not intuitive to enter the HxW resolution, moreover this doesn't care about maintaining the aspect ration. 

but my primary question remains the same, what is the need for this. 

the rest of the change look good, I'll go ahead with those, I want to hold back on this.